### ISSUE TITLE
Accessibility issues on Environment Window: Combobox

### ISSUE THREAD

**Issue by eclectir** [Microsoft/PTVS#2226]
![image](https://cloud.githubusercontent.com/assets/5177491/23570659/1582310c-001a-11e7-9a2b-dea25738c0c5.png)

**Comment by eclectir** [comment]
Error details-
MissingProperty - UIA_NameProperty - Element has no accessible name. If a user reaches an element with no name, then often they will have no way of knowing what the element relates to.
https://msdn.microsoft.com/en-us/library/system.windows.automation.automationelement.automationelementinformation.name(v=vs.110).aspx
